.travis.yml

```diff
@@ -22,8 +22,23 @@ script:
   - bin/test.sh
   - tests/tasks.sh
 
-
-matrix:
-  fast_finish: true
-  allow_failures:
-    - rust: nightly
+env:
+  global:
+    - PROJECT_NAME=rumake
+    - TARGET=x86_64-unknown-linux-gnu
+
+before_deploy:
+  - mkdir releases
+  - cargo build --release --target=$TARGET
+  - strip target/${TARGET}/release/${PROJECT_NAME}
+  - tar czf releases/${PROJECT_NAME}-${TRAVIS_TAG}-${TARGET}.tar.gz README.md -C target/${TARGET}/release/ ${PROJECT_NAME}
+
+deploy:
+  provider: releases
+  api_key:
+    secure: yIJ6jiYwjCjk3lVUIOeZcrcDTsjv+8suK/bpqRPW0lKDMtw8m89OJFNchIxY0eMmJ12HQDT0PuYafCOHV/HcJrDcepY8vyq0LSpgKrXv7pqmIHxRS0BQUfmJnIMRJKwyakwidtp5+tnlMMmtdJM5S1X+XVOmWMSFgw2nmm1s/viJ0/1FG0iR6XGhPeKUKoHtOMsQVQjR7j6iSjnnXvwHJVlAd/TdU15oBjGbi2hydjB3i1nO0cFozcqoKX2scu3E/gszR2HRnykv0Zu5KaKft7GOpF27ZA5bzsXRHKrCEbmFJ1+2zsG0/ube7aUj+TXOamYoFimgiIxXNxeNd6MSSxlh+bPnJBQskBt6VxZFM/EfHYEB+b30TiHAUkOFK4Wq1gZxvXZ+xIujn9M7ajImAN4F1zBQ8V3vsYqRpDvi3wSdfnUTPtQ0yZp+szJDQhZEkJRzp+ApbzPxuehsCxV6fnhnDqVZ/9sNB3PEzU+iG4cG7AO3T+ypx2AzwMWXWb3yaSO+5cVAmSbeBpRsl15agIjT/2YQ9hZQ+mVW/MiPYbV1WAYSul+nNIUWE0t/b8g/p0JBpv54IZggnStwYj6MXonlgFOb2MoWlGzsY221DSzZ6nGUo4uzSZi7Twhsm36y8du+3HUun9Ns9iBgng6s1SyqXsjAdSmm5uhRDEF9BUA=
```

Comment:
Is it intended to store this publicly ?

Reply:
Afaik it is. You can find more information here https://docs.travis-ci.com/user/deployment/releases/#authenticating-with-an-oauth-token
Do you think of a better way to do it ?

Reply:
Nope